CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - fix: SentryOptions.tracesSampleRate default value (#1069)
 - ref: Discard unfinished spans when capturing transaction (#1066)
 - ref: Make calls to customSamplingContext nonnull (#1061)
+- fix: fix: performance headers (#1079)
```

Comment:
👀

Reply:
Fixed with https://github.com/getsentry/sentry-cocoa/commit/e14fa827cc83c0cb0e0dd9f2257d510654c000ea

Reply:
I had to read it twice to notice it